dask/dataframe/tests/test_io.py

```diff
@@ -547,11 +547,10 @@ def test_read_csv_of_modified_file_has_different_name():
     with filetext(text) as fn:
         mtime = os.path.getmtime(fn)
         a = read_csv(fn)
+        sleep(1)
         with open(fn, 'a') as f:
             f.write('\nGeorge,700')
             os.fsync(f)
-        while mtime == os.path.getmtime(fn):
-            sleep(0.01)
```

Comment:
So, as someone who works on dask a lot and runs tests on every file save, I do find one second to be a pretty big pain.  Consider this my equivalent of pep8 nit-pickyness.

Currently dask.dataframe tests run on my machine in 8s and I might go through and speed up these tests in the near future (they're feeling quite bloated now.)  Ideally I like to stay within 1-3s for any particular module that I work on if possible.  There are obviously cases where this can't be done (e.g. distributed) but even then we can be less-than-terrible.